options/locale/locale_de-DE.ini

```diff
@@ -1147,6 +1147,10 @@ settings.pulls.allow_squash_commits=Mergen von Commits durch Squash aktivieren
 settings.admin_settings=Administratoreinstellungen
 settings.admin_enable_health_check=Repository-Health-Checks aktivieren (git fsck)
 settings.admin_enable_close_issues_via_commit_in_any_branch=Einen Issue mit einem Commit auf einem nicht-Standard-Branch schließen
+settings.toc.toc = Inhaltsverzeichnis
```

Comment:
Please don't change de-DE translation file in source code but translate it on crowdin.

Reply:
Removed